python/profiler.cpp

```diff
@@ -0,0 +1,56 @@
+#include <pybind11/pybind11.h>
+#include <sstream>
+
+#include <arbor/profile/meter_manager.hpp>
+#include "context.hpp"
+
+namespace arb {
+namespace py {
+
+void register_profilers(pybind11::module& m) {
+    //
+    // metering
+    //
+    pybind11::class_<arb::profile::meter_manager> meter_manager(m, "meter_manager");
+
+    meter_manager
+        .def(pybind11::init<>())
+
+        // Need to use shimming of context due to pybind11 complaining about incomplete type of execution_cntext  
+        .def("start", [](arb::profile::meter_manager& manager, const arb::py::context_shim& ctx){\
+            manager.start(ctx.context);\
```

Comment:
You don't need the `\` on the end of split lines in C++. It is only every used when splitting strings over multiple lines.